client/js/lounge.js

```diff
@@ -141,6 +141,12 @@ $(function() {
 					text: "Join a channel…",
 					data: target.data("id"),
 				});
+				output += templates.contextmenu_item({
+					class: "edit",
+					action: "edit",
+					text: "Edit network",
+					data: target.closest(".network").data("uuid"),
+				});
```

Comment:
Mind putting this right after the network name, and calling it "Edit this network..."?